.github/workflows/test-check-tos.yml

```diff
@@ -0,0 +1,16 @@
+name: TEST - Gradle - Terms of Service approval verification
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+  actions: write
+  statuses: write
+
+jobs:
+  gradle-check-tos:
+    uses: ./.github/workflows/check-tos.yml
+    with:
```

Comment:
I don't understand how this works without specifying `secrets` here. https://docs.github.com/en/actions/using-workflows/reusing-workflows#passing-secrets-to-nested-workflows

Reply:
Is this clarified with the fact that the `GITHUB_TOKEN` is passed implicitly?